splitio/client/factory.py

```diff
@@ -189,7 +201,15 @@ def block_until_ready(self, timeout=None):
             ready = self._sdk_ready_flag.wait(timeout)
 
             if not ready:
+                self._telemetry_init_producer.record_bur_time_out()
```

Comment:
this should be record non ready usage, the bur timeouts is when the exception of timeout is raised

Reply:
The exception line is below it though, so it is raising the timeout exception.